package.json

```diff
@@ -3,13 +3,14 @@
   "version": "1.0.0",
```

Comment:
Could update the version now.

Reply:
Why?
This PR do not change the content:
  * do not fix any bug
  * do not add backward compatible functionality
  * do not change the API

Npm packages are expected to follow [semantic versioning](https://docs.npmjs.com/getting-started/semantic-versioning).
See [semver.org summary](https://semver.org/#summary) .

Reply:
It does change the content (e.g. README), otherwise we can't release that.

Reply:
As [this stackoverflow answer's comments](https://stackoverflow.com/questions/13302995/version-bump-when-updating-readme-for-npm-package#13369141) state, indeed, it's needed to bump the patch version to update the `README.md`.